src/main/java/gov/nasa/jpl/mbee/mdk/mms/actions/CommitClientElementAction.java

```diff
@@ -121,8 +121,6 @@ private static void request(List<ObjectNode> elementsToUpdate, List<String> elem
             try {
                 File file = MMSUtils.createEntityFile(CommitClientElementAction.class, ContentType.APPLICATION_JSON, elementsToUpdate, MMSUtils.JsonBlobType.ELEMENT_JSON);
                 URIBuilder requestUri = MMSUtils.getServiceProjectsRefsElementsUri(project);
-                // Prevent potential "Not Equivalent" errors due to default merging of element data.
-                requestUri.addParameter("overwrite", "true");
```

Comment:
I think leaving this in would be fine, since I believe this is a user manual action. Is there another action or option for choosing overwrite? Not sure if the user would understand the implications though. (I'm fine with reverting this, but not sure if having a way to solve these errors is a priority for the release) @ivan-gomes

Reply:
There is currently only one user action for "Commit Element to MMS" without variants. I think we should revert it for now; having a solution for this specific problem is not a priority for the release.

Some things to consider when this is being revisited:
* presentation elements manually added via View Editor must be preserved, so simply wiping `_contents` as committing with `overwrite=true` would do and then triggering regeneration with DocGen will not suffice as it would be a loss of data
* The state of `_contents` is only maintained in MMS - it cannot be computed by model data alone